### DISCUSSION TITLE
connection monitoring

### DISCUSSION THREAD

**Discussion by Smiley-k** [rustdesk/rustdesk#5881] [Q&A]
Hi, can you please tell me how I can monitor the connections that are made to my server ? 

I would like to receive from the database information on all my IDs (devices) current status (connected or disconnected from the server) time of the last connection and time of disconnection.

For what ? I have a dashboard on which I display all the connectivity, so that I do not have to wait until I am informed that at the other end of the rustdesk or turn on the PC on which rustdesk is running.

So in would like to display whether my home PC is available for management, whether my server is available for management and so on, how to do it with Rustdesk? 

Thank you.

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/

**Reply by Smiley-k** [reply]
Well, that's not fair ).... And seriously, I'm not talking about internal monitoring inside the application, but getting data from the database itself..... Even if I buy the pro version, how can I get the status of connected agent to the server by my monitoring system?
